src/scripts/components/NoticesLoop.js

```diff
@@ -4,10 +4,10 @@ import { Notice } from './Notice';
  * Returns a list of notices
  * each notice is a component that has a key, an id, an image, an additional class name, and an onDismiss function
  *
- * @param {Object} prop
- * @param {Array}  prop.notices - An array of objects that contain the notice data.
+ * @param {Object}                      prop
+ * @param {import('../store').Notice[]} prop.notices - An array of objects that contain the notice data.
```

Comment:
There is a name conflict if this is imported in a typedef, it could be renamed in this one instance, but I don't really like that, it looks like a different type in function signature.

```js
/**
 * @typedef {import('../store').Notice} Notification 
 */
```